### PR TITLE
Contrib

### PR DESCRIPTION
**Pull request by wwlytton**
New page to documentation to encourage and then list user contribution of ancillary (mostly analysis) flies.

### CODE REVIEW

File: doc/source/about.rst [hunk 1]
```diff
@@ -88,6 +88,8 @@ Major Features
 Questions, suggestions, and contributions
 -----------------------------------------
 
+Two kinds of contributions are solicited: 1. direct contributions to NetPyne code described below. 2. `Ancillary code contributions <contrib.html>`_.
```
Comment (salvadord) [inline doc/source/about.rst:91]: NetPyNE

File: doc/source/index.rst [hunk 2]
```diff
@@ -17,6 +17,8 @@ Check out our latest tutorial: **Virtual Environments, Jupyter Notebooks, and Ne
 
 Join our `NetPyNE mailing list <https://groups.google.com/forum/#!forum/netpyne-mailing>`_ to receive updates on version releases and other major announcements.  Get your questions answered and participate in the NetPyNE community at the `NetPyNE forum <https://groups.google.com/forum/#!forum/netpyne-forum>`_ or the `NetPyNE section of the NEURON forum <https://www.neuron.yale.edu/phpBB/viewforum.php?f=45>`_.
 
+NetPyNE is available as a resource through `Neuroscience Gateway Portal (NSG) <https://www.nsgportal.org/>`_;  `Open Source Brain (OSB) <https://www.opensourcebrain.org/>`_ ; `Human Brain Project (HBP) eBrains <https://www.humanbrainproject.eu/en/hbp-platforms/ebrains/>`_; `NIH SPARC initiative <https://commonfund.nih.gov/Sparc/>`_.
```
Comment (salvadord) [inline doc/source/index.rst:20]: eBrains -> EBRAINS

Comment (salvadord) [reply doc/source/index.rst:20]: ebrains link -> https://ebrains.eu/services/simulation

**Comment by wwlytton** [comment]
fixed

**Comment by wwlytton** [comment]
added list of NetPyNE users; added new page for contrib ancillary code

**Pull request merged**